src/openalea/strawberry/application/app_visualization.py

```diff
@@ -44,10 +44,26 @@ def print_3d_floral_intensity(mtg, genotype):
             print('Select a Genotype') 
 
 
-def print_2d_single_p():
+def print_2d_single_p(mtg, genotype, vid):
```

Comment:
I often use `g` for MTG variable rather than `mtg`. This allow to avoid name clash with mtg pkg.